php/example_code/lambda/Readme.md

```diff
@@ -0,0 +1,98 @@
+# AWS Lambda code examples for the SDK for PHP
+
+## Overview
+
+Shows how to use the AWS SDK for PHP to create, deploy, and invoke
+AWS Lambda functions. Learn to accomplish the following tasks
+
+*Lambda lets you run code without provisioning or managing servers. Upload your code
+and Lambda takes care of everything required to run and scale your code with high
+availability.*
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single action
+
+* [Create a function](LambdaService.php)
+  (`CreateFunction`)
+* [Delete a function](LambdaService.php)
+  (`DeleteFunction`)
+* [Get a function](LambdaService.php)
+  (`GetFunction`)
+* [Invoke a function](LambdaService.php)
+  (`Invoke`)
+* [List functions](LambdaService.php)
+  (`ListFunctions`)
+* [Update function code](LambdaService.php)
+  (`UpdateFunctionCode`)
+* [Update function configuration](LambdaService.php)
+  (`UpdateFunctionConfiguration`)
+
+### Scenarios
+
```

Comment:
Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.

(description from template)

Reply:
fixed